app/controllers/chargeback_controller.rb

```diff
@@ -205,25 +205,22 @@ def cb_rates_delete
     if !params[:id] # showing a list
       rates = find_checked_items
       if rates.empty?
-        render_flash(_("No Chargeback Rate were selected for deletion"), :error)
+        add_flash(_("No Chargeback Rates were selected for deletion"), :error)
       end
     else # showing 1 rate, delete it
-      cb_rate = ChargebackRate.find_by_id(params[:id])
+      cb_rate = ChargebackRate.find_by(:id => params[:id])
+      self.x_node = x_node.split('_').first
```

Comment:
@hstastna same here, i think we don't need to reset value of x_node if cb_rate is nil

Reply:
I needed this for `cb_rates_list` and `replace_right_cell` methods which come after this block.